src/leiningen/new/options/scss.clj

```diff
@@ -0,0 +1,7 @@
+(ns leiningen.new.options.scss
+  (:require [leiningen.new.options.helpers :as helpers]))
+
+(def option "scss")
```

Comment:
This needs to be `(def option "+scss")` (i.e. missing `+`). Fixes `Error: invalid profile(s)`.